src/main.cpp

```diff
@@ -1,7 +1,7 @@
-#include <iostream>
-
-int main() {
-    std::cout << "Hello, world!" << std::endl;
-
-    return 0;
-}
+#include <CSGServer/main.hpp>

```

Comment:
Project headers must be quoted by double quotes. See [Code Style](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes).

Reply:
Also, rename branch to `feature/1-folder-structure`.